File: sprint1/By languages/kotlin/lesson7code3.kt
fun getSmallerPrimes(n: Int): List<Int> {
    val smallerPrimes = mutableListOf<Int>()
    for (num in 2..n) {
        if (isPrime(num)) {
            smallerPrimes.add(num)
        }
    }
    return smallerPrimes
}

File: sprint1/By languages/C#/lesson7code3.cs
public static List<int> GetSmallerPrimes(int n)
{
    List<int> smallerPrimes = new List<int>();

    for (int num = 2; num <= n; num++)
    {
        if (IsPrime(num))
        {
            smallerPrimes.Add(num);
        }
    }

    return smallerPrimes;
}
